src/physics/cam/clubb_intr.F90

```diff
@@ -2942,6 +2973,11 @@ subroutine clubb_tend_cam( &
       rtp2_zt = zm2zt_api(rtp2_in)
       thl2_zt = zm2zt_api(thlp2_in)
       wp2_zt  = zm2zt_api(wp2_in)
+!+++MKW
+      ! Need moist_qc and cloudfrac on thermo grid for output
+      mf_qc_zt = zm2zt_api(mf_moist_qc)
+      mf_cloudfrac_zt = zm2zt_api(mf_cloudfrac)
+!---MKW

```

Comment:
Another double-check, I'm pretty sure these interpolations don't introduce new minima/maxima, which means they can't create negative values, or cloudfracs > 1. I should probably double-check this.

Reply:
verified it's linear:

  logical, parameter, public :: &
    l_cubic_interp = .false.      ! Flag to convert grid points with cubic monotonic
                                  ! spline interpolation as opposed to linear interpolation.